mkdocs.yml

```diff
@@ -253,8 +253,8 @@ nav:
     - Tooltips: reference/tooltips.md
   - Insiders:
     - insiders/index.md
-    - Getting started:
-      - Installation: insiders/getting-started.md
-      - Changelog: insiders/changelog.md
+    - Getting started with Insiders: insiders/getting-started.md
+    - Changelog: insiders/changelog.md
```

Comment:
Could we put this into a section, similar to how we have it in our setup guide?

``` yaml
nav:
  ...
  - Changelog:
    - insiders/changelog/index.md # <- we'd need to rename the file, but URLs should stay the same
    - How to upgrade: insiders/upgrade.md
```

The structure would then be identical.

Reply:
Ah! Yes, we need to adjust the shortcodes hook as well – good catch!

https://github.com/squidfunk/mkdocs-material/blob/b77769009ece2326476d1a02d8fa04e501f73af0/src/overrides/hooks/shortcodes.py#L148